packages/styles/src/components/datatable.scss

```diff
@@ -67,22 +68,18 @@
   width: spacing.$size-regular;
   height: spacing.$size-regular;
   margin-left: spacing.$size-mini;
-  background-repeat: no-repeat;
-  background-size: contain;
 
   &.datatable-icon-sort-unset {
-    background-image: datatable.$datatable-sort-unset-icon;
-    background-position: center top, center bottom;
+    @include icons-mx.icon(2127);
   }
 
-  &.datatable-icon-up {
-    background-image: datatable.$datatable-sort-asc-icon;
-    background-position: center center;
+  &.datatable-icon-up,
+  &.datatable-icon-down {
+    @include icons-mx.icon(2112);
```

Comment:
What do you think, should we document this mixin on the icons docs page?

Reply:
I don't think this mixin should be used externally.
We use it internally because styles need to display icons without being able to modify the HTML structure, but this is not the case for Design System users who need to use the post-icon component.